core/src/docs/rfcs/mod.rs

```diff
@@ -15,166 +15,204 @@
 // specific language governing permissions and limitations
 // under the License.
 
-//! RFCs power OpenDAL's development.
-//!
-//! To add new features and big refactors:
-//!
-//! - Start a new RFCs with the template [`rfc_0000_example`].
-//! - Submit PR and assign the RFC number with the PR number.
-//! - Request reviews from OpenDAL maintainers.
-//! - Create a tracking issue and update links in RFC after approval.
-//!
-//! Some useful tips:
-//!
-//! - Start a pre-propose in [discussion](https://github.com/apache/incubator-opendal/discussions/categories/ideas) to communicate quickly.
-//! - The proposer of RFC may not be the same person as the implementor. Try to include enough information in RFC itself.
+#![doc = include_str!("README.md")]
 
+/// RFC example
 #[doc = include_str!("0000_example.md")]
 pub mod rfc_0000_example {}
 
+/// Object native API
```

Comment:
This will add extra work to add new RFCs, how about removing them?

Reply:
This is also to enhance the aesthetics of our RFC list simply. 
I am open to doing the additional work, even if it involves just one line.